pets/mysql/galera/README.md

```diff
@@ -0,0 +1,26 @@
+# Mysql Galera
+
+This example runs mysql galera through a petset.
+
+## Bootstrap
+
+Create the petset in this directory
+```
+$ kubectl create -f galera.yaml
+```
+
+This example requires manual intervention. Starting up all galera nodes at once
```

Comment:
What's the actual manual intervention?  Could you emulate that in the on-start script?